### ISSUE TITLE
`.method()` should be placed on a new line in method chaining, even when the prevous line ends with a `)` or `}`

### ISSUE THREAD

**Issue by testersen** [pinterest/ktlint#2899]
## Expected Behavior

In method chaining, I'd expect the `.method()` to appear on a new-line and not after a `)` or `}`.

```kotlin
myObject
    .method1(
        mapOf(
            "a" to "b",
            "c" to "d",
        )
    )
    .method2()
```

## Current Behavior

Currently the `.method()` is placed directly after `)` and `}` for example:

```kotlin
myObject
    .method1(
        mapOf(
            "a" to "b",
            "c" to "d",
        )
    ).method2()
```

## Why?

Placing method calls (e.g., .method()) directly after closing parentheses ) or braces } introduces unnecessary friction when modifying method chains. Specifically, it requires developers to precisely position the cursor between the closing token and the dot (.) to insert a new line for adding or rearranging method calls. This can disrupt workflow, especially for those accustomed to using keyboard shortcuts for line manipulation, such as moving lines up or down or inserting new lines in between existing lines.

When method calls are aligned on new lines, the development experience is significantly improved:
1. Streamlined Edits: Developers can easily use keybindings to move entire lines up or down or insert new method calls between existing ones without fiddling with cursor placement.
2. Reduced Frustration: Avoiding the need to carefully navigate to specific positions between tokens and dots eliminates small but frequent interruptions in focus.
3. Improved Readability: Clear vertical alignment of method calls enhances code clarity, making chains easier to follow and understand.
4. Better Maintainability: Changes to method chains are less disruptive and more intuitive, encouraging clean and frequent refactoring.

Aligning method calls on new lines not only adheres to principles of clean code but also respects developers’ workflows, minimizing pain points and enabling a smoother editing experience.

## Additional information

- Current version of ktlint: 1.4.1

**Comment by testersen** [comment]
@paul-dingemans I have tried disabling / suppressing through annotations, but this has not worked.

**Comment by paul-dingemans** [comment]
Please provide code sample and config that you tried. Probably you just made a small error.
